widget/ui/src/containers/SwapInput/SwapInput.tsx

```diff
@@ -72,36 +78,43 @@ export function SwapInput(props: SwapInputProps) {
             </>
           ) : (
             <>
-              <InputAmount
-                disabled={props.disabled || props.mode === 'To'}
-                style={{ padding: 0 }}
-                value={props.price.value}
-                type={'onInputChange' in props ? 'number' : 'text'}
-                size="large"
-                placeholder="0"
-                variant="ghost"
-                min={0}
-                {...('onInputChange' in props && {
-                  onChange: (event: React.ChangeEvent<HTMLInputElement>) =>
-                    props.onInputChange(event.target.value),
-                })}
-              />
+              <Tooltip
+                content={props.price.realValue}
+                open={props.mode === 'From' ? false : undefined}>
+                <InputAmount
+                  disabled={props.disabled || props.mode === 'To'}
+                  style={{ padding: 0 }}
+                  value={props.price.value}
+                  type={'onInputChange' in props ? 'number' : 'text'}
+                  size="large"
+                  placeholder="0"
+                  variant="ghost"
+                  min={0}
+                  {...('onInputChange' in props && {
+                    onChange: (event: React.ChangeEvent<HTMLInputElement>) =>
+                      props.onInputChange(event.target.value),
+                  })}
+                />
```

Comment:
It would be preferable to hide the tooltip if the token amount or USD value is 0 :
![image](https://github.com/rango-exchange/rango-client/assets/120931880/78037abf-d956-435c-931f-9db4af8cd9e9)

Reply:
fixed.